testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,84 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
 
     protected static final THIRD_PARTY_LIB_COUNT = 144
 
-    @Shared String baseVersion = GradleVersion.current().baseVersion.version
+    @Shared
+    String baseVersion = GradleVersion.current().baseVersion.version
+
+    def coreLibsFileNames = [
```

Comment:
💭  I would call these `coreLibsModules` and have a list of their plain names here, such as `base-asm`, `base-services`, etc.
Then I would do necessary prefixing/suffixing in the test.

I'm also okay with keeping the `gradle-` prefix here, but then let's call it `coreLibJars`.

Reply:
I think we should the `-` suffix as well, so the list reads nicely